basic-sql-operations.md

```diff
@@ -1,174 +1,156 @@
 ---
-title: Explore SQL with TiDB
+title: Basic SQL operations
 summary: Learn about the basic SQL statements for the TiDB database.
 aliases: ['/docs/dev/basic-sql-operations/','/docs/dev/how-to/get-started/explore-sql/']
 ---
 
-# Explore SQL with TiDB
+# Basic SQL Operations
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most of the cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+Structured Query Language (SQL) is a declarative language which defines the way that users interact with databases. You can use SQL just as a natural language to talk to the database. This page walks you through the basic SQL statements in TiDB. For a complete list of TiDB statements, see [TiDB SQL Syntax Diagram](https://pingcap.github.io/sqlgram/).
```

Comment:
For this section, I suggest using the existing English version. @ireneontheway @ran-huang If the existing English version is better, just keep them.